packages/liveblocks-core/src/client.ts

```diff
@@ -198,7 +198,7 @@ export function createClient(options: ClientOptions): Client {
         global.atob = clientOptions.polyfills.atob;
       }
 
-      newRoom.__internal.connect();
+      newRoom.__internal.simulate.connect();
```

Comment:
Did you mean to call `handleConnect` here and not simulate? This is the actual client right?

Reply:
Agreed, this name change was made mainly from a unit test perspective. The pattern here is "sending messages to the room's internals", so it can act on it. Of course, from where we use this in the actual client, this isn't _simulating_, but _actually doing_. So I agree the naming is bad here.

Perhaps I can name this group `send`? So it reads as:

```
newRoom.__internal.send.connect()
```

That way, it may be less confusing?

Either way, what's important here is the grouping itself (independent of the name), which I've done to more clearly see which parts are for "writing to" vs "reading from" the room's internals. I'll soon replace this code with a much cleaner API anyway! (Working on that still.)

Reply:
Rebased + fixed the name in f3c71caf63870221b5fc169521554dd89661cbbe. Hope this is more clear?

Reply:
yes, thank you!